### ISSUE TITLE
Missing some examples/tutorial

### ISSUE THREAD

**Issue by Yousha** [maddinat0r/samp-discord-connector#142]
Having some usage examples or tutorials would be great.

**Comment by DERFMAX** [comment]
The exact same thing happens to me, I have done everything that comes to my mind but I don't know how to do it. If someone can help me I would be so thankful

**Comment by WiltonFreitas** [comment]
@Alasnkz can you create a some wiki with examples to the embed functions?

**Comment by Alasnkz** [comment]
Yes, I will work on tutorials and such after I release the next update.
